### PR TITLE
Lesson-1-4

### PR DESCRIPTION
**Pull request by Armanchi**

### CODE REVIEW

File: src/AddTodoForm.js [hunk 1]
```diff
@@ -1,13 +1,35 @@
...
+    const [todoTitle, setTodoTitle] = useState('');
+    const handleTitleChange = (e) => {
+        const newTodoTitle = e.target.value
+        setTodoTitle(newTodoTitle)
+    }
+    const handleAddTodo = (event) => {
+        event.preventDefault();
+        console.log(todoTitle);
+        onAddTodo({title: todoTitle}, {id: Date.now()});
+        event.target.title.value = ''
```
Comment (kramduckner) [inline src/AddTodoForm.js]: The value reset isn't working : (. You'll have to use your new state setting function!

File: src/TodoList.js [hunk 2]
```diff
@@ -1,35 +1,20 @@
...
-    )
 
-}
+const TodoList = props => {
+  const {todoList} = props;
+	return (
+		<div>
+        <ul >
+				{todoList.map((todo) => {
+          return <TodoListItem todo={todo} />
```
Comment (kramduckner) [inline src/TodoList.js]: This isn't necessary for functionality, but adding a key attribute on any mapped element is a good practice for performance! If you run the app in a browser and open the console, you'll see a warning message about this. It'd be awesome if you'd update the code so the warning message no longer appears!

File: src/App.js [hunk 3]
```diff
@@ -1,15 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
 
-function App() {
+
+const App = () => {
+  const [todoList, setTodoList] = useState([]);
+  const addTodo = (newTodo) => {
```
Comment (kramduckner) [inline src/App.js:8]: If you run the app in the browser and console log the newTodo argument, you'll see that there's no id property in the object - do you mind fixing that?

**Review by kramduckner — CHANGES_REQUESTED** [review]
Awesome job! It's working really well and it's obvious you have a good grasp of some of the harder topics in React. Just a few requests.

File: src/AddTodoForm.js (hunk 1 above)
Comment (Armanchi) [reply src/AddTodoForm.js]: I made some changes! Thank you :)

**Review by kramduckner — APPROVED** [review]
Approved! Thanks for making those changes!

**Pull request merged**